### PR TITLE
[new modeling strategy] maternal supplementation effect on birthweight exposure

### PR DESCRIPTION
**Pull request by alibow**

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]

File: docs/source/intervention_models/targeted_bep_and_mms/index.rst
Comment (yongqx) [inline docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: @alibow Is that means the birthweight shift = +57.73g if someone takes IFA; and +(57.73 + 45.16)g for MMN, and +(57.73 + 45.16 + 66.96)g for BEP; assuming all three groups compare to no supplementations.

Comment (alibow) [reply docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: Yep, exactly!

Comment (alibow) [reply docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: @yongqx, can you please approve this PR if you have finished reviewing? I can merge all of the supplementation PRs after this one is approved. Thanks!

**Review by yongqx — APPROVED** [review]

File: docs/source/intervention_models/targeted_bep_and_mms/index.rst
Comment (alibow) [reply docs/source/intervention_models/targeted_bep_and_mms/index.rst:220]: Thank you!

**Pull request merged**
